server/controllers/finance/vouchers.js

```diff
@@ -0,0 +1,152 @@
+/**
+* The /vouchers/ HTTP API endpoint
+*
+* @module finance/vouchers
+*
+* @desc This module is responsible for handling all crud operations relatives
+* to voiuchers transactions and define all vouchers api functions
+*
+* @required node-uuid
+* @required lib/db
+*/
+
+'use strict';
+
+var _    = require('lodash'),
+    uuid = require('node-uuid'),
+    db   = require('../../lib/db');
+
+/** Get list of vouchers */
+exports.list = getList;
+
+/** Get detail of vouchers */
+exports.detail = getDetail;
+
+/** Create a new voucher record */
+exports.create = create;
+
+/**
+* GET /vouchers
+*
+* @method getList
+*/
+function getList(req, res, next) {
```

Comment:
Why not just call this method `list()` so that it looks like the exposed module API?


Also, the `list()` methods should return only an id and a name -- if you need more information, that should be behind a query string flag.  We are trying to limit the size of our HTTP requests.